src/pharmpy/tools/resmod/tool.py

```diff
@@ -133,12 +133,14 @@ def _create_dataset(input_model):
 
 def _create_best_model(model, res):
     model = model.copy()
-    if any(res.models['dofv'] > 3.84):
-        idx = res.models['dofv'].idxmax()
+    if any(-res.models['dofv'] > 3.84):
+        idx = res.models['dofv'].idxmin()
         name = idx[0]
         if name == 'power':
             set_power_on_ruv(model)
+            model.parameters.inits = {'power1': res.models['parameters'].iloc[1].get('theta')}
```

Comment:
`iloc[1]` is dangerous since it might change in the future. Instead use the row-index of the `res.models`.  For example: `res.models['parameters'].loc['power', 1, 1]`